playground/README.md

```diff
@@ -0,0 +1,66 @@
+# SignIn Widget Playground
+
+## The playground App
+In a nutshell, the playground application demonstrates customer hosted sign-in widget case, which loads sign-in widget assets and bootstrap widget via configurations.
+The configurations have been done in two folds
+
+- [widgetrc](https://github.com/okta/okta-signin-widget#the-widgetrc-config-file)
+- `./main.js`
+
+What `main.js` does is initialize sign-in widget using settings from `widgetrc` plus simple success handler.
+
+## Mock server
+
+While in `widgetrc` you can point `baseUrl` to any Okta tenants, it can also be set to mock server, which is started at `http://localhost:3000` when starting playground application.
+You could imagine this is just another Okta tenant but it has to handle API requests properly on its own, which of course means mock it.
+
+Here is the directory structure of `mocks` folder
+
+```
+playground/mocks
+├── config
+├── data
+├── spec-device-authenticator
+└── spec-okta-api
+```
+
+The mock server is implemented using [dyson.js](https://www.npmjs.com/package/dyson). Basically, you need to create a *mapping* in terms of what URI shall respond what response and those *mappings* are all living in `spec-*` folders.
+
+- `spec-okta-api` has mocks for Okta APIs, e.g. `/api/v1/*`, `/idp/idx/*`, etc
+- `spec-device-authenticator` is for mocking 3rd-party device authenticator application.
+
+Most of cases, the response is from a JSON file, it would be good to have abstraction to avoid tedious copy/parse. Take a look at `config/templateHelper` regardling how to handle the common use in general fashion.
```

Comment:
few typos/corrections: 
In most cases, the response is coming from a JSON file. It would be good to have abstractions to avoid tedious copy/paste. Take a look at `config/templateHelper` to get a sense of how to handle the common use cases in a generic fashion.